test/test_nn_quantized.py

```diff
@@ -34,6 +36,67 @@ def test_relu_api(self):
 
 
 class ModuleAPITest(TestCase):
+    @given(
+        batch_size=st.integers(1, 5),
+        in_features=st.integers(16, 32),
+        out_features=st.integers(4, 8),
+        use_bias=st.booleans(),
+    )
+    def test_dynamic_linear_api(self, batch_size, in_features, out_features, use_bias):
+        """test API functionality for nn.quantized.dynamic.Linear"""
+        W = torch.rand(out_features, in_features).float()
```

Comment:
FYI: I saw `rand` outside the `hypothesis` sometimes makes the tests "flaky". No need to change anything, but in case you notice falkiness, this might be causing it